### PR TITLE
Feature/add job board

### PR DESCRIPTION
**Pull request by bhansa**
- fetch data from google form response sheet
- fetch only every 2hours

### CODE REVIEW

File: src/jobs.liquid [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+layout: content.liquid
+---
+
+<section class="container pt-md-4 pb-md-2" id="job-board">
+  <h1 class="text-center">Job Board</h1>
+  <div class="jobs-container">
```
Comment (SaptakS) [inline src/jobs.liquid]: This should be a `<ul>`

File: src/jobs.liquid [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  {% if jobs.length > 0 %}
+    {% for job in jobs %}
+      <div class="col-12 d-flex mb-lg-0 mb-4 mt-4 p-2">
```
Comment (SaptakS) [inline src/jobs.liquid]: This should be `<li>`

File: src/jobs.liquid [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        <div class="jobcard card shadow p-4 rounded-md">
+          <li class="d-flex flex-column">
```
Comment (SaptakS) [inline src/jobs.liquid]: I don't think this and the rest of the `<li>`s below should be `<li>`. They can just be simple `div`I think.

File: src/jobs.liquid [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+              <h4 class="text-dark">{{job.jobTitle}}</h4>
```
Comment (SaptakS) [inline src/jobs.liquid]: This should be a `h2` not `h4`. Else there will be a jump in heading order.

File: src/jobs.liquid [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+          </li>
+
+          <li class="d-flex flex-column">
+            <a href="{{job.companyWebsite}}" class="talk-link">
+              <h5>{{job.companyName}}</h5>
```
Comment (SaptakS) [inline src/jobs.liquid]: This should be a `h2` not `h5`. Else there will be a jump in heading order. For styling, you can add class `.h5`

**Review by SaptakS — CHANGES_REQUESTED** [review]
Requested few HTMl semantic changes.

**Comment by anistark** [comment]
@bhansa please add a button to the google form to add jobs as well?

File: src/_data/jobs.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
+const EleventyFetch = require("@11ty/eleventy-fetch");
+
+module.exports = async function () {
+  async function fetchSheetData(sheetId, sheetIndex) {
+    try {
+      const url = `https://api-sheets-bhansa.netlify.app/.netlify/functions/fetchSheetsData?sheetId=${sheetId}&sheetIndex=${sheetIndex}`;
```
Comment (palnabarun) [inline src/_data/jobs.js:6]: where's the code of the function being hosted?

Comment (palnabarun) [reply src/_data/jobs.js:6]: We can easily host it in our server. I don't want to depend on Netlify for this.

**Review by palnabarun — CHANGES_REQUESTED** [review]

File: src/_data/jobs.js (hunk 6 above)
Comment (bhansa) [reply src/_data/jobs.js:6]: Sure, if you're aware and comfortable with other options, do try to host it on your server and switch to it once done. Whatever feels right to you.
Meantime, we can continue to use this.
**Note**: The server only fetches the data from google sheets using a backend service, the code is open and here the repository https://github.com/bhansa/api-sheets-data.

Comment (palnabarun) [reply src/_data/jobs.js:6]: In the interest of time, let's continue with the current approach for now.

**Review by palnabarun — APPROVED** [review]

**Review by palnabarun — APPROVED** [review]

**Review by SaptakS — APPROVED** [review]

**Pull request merged**